assets/js/state/sagas/operations.test.js

```diff
@@ -27,96 +34,163 @@ describe('operations saga', () => {
     console.error.mockRestore();
   });
 
-  it('should request an operation', async () => {
-    const groupID = faker.string.uuid();
-    const operation = faker.lorem.word();
-    const hostname = faker.internet.displayName();
-
-    axiosMock.onGet(hostOperationRequestURL(groupID)).reply(202, {});
-
-    const dispatched = await recordSaga(
-      requestOperation,
-      {
-        payload: { groupID, operation },
-      },
-      { hostsList: [{ id: groupID, hostname }] }
-    );
-
-    expect(dispatched).toContainEqual(
-      setRunningOperation({ groupID, operation }),
-      notify({
-        text: `Operation ${operation} requested for ${hostname}`,
-        icon: '⚙️',
-      })
-    );
+  describe('request operation', () => {
```

Comment:
It looks like there are many changes, but I simply grouped existing tests in their own describes and added the new `describe` for `update running operation` tests.